projectile.el

```diff
@@ -502,7 +507,35 @@ just return nil."
         version)
     (error "Cannot determine version without package pkg-info")))
 
-
+;;; Workarounds
+
+(eval-after-load 'etags
```

Comment:
I think there should be a comment here explaining the need for this. Btw, we already have a place in the code where we're backporting stuff from newer Emacsen.